engine/src/options.cpp

```diff
@@ -411,3 +411,8 @@ string vs_options::getPassword(int playernum) {
             "password",
             ""));
 }
+
```

Comment:
Why did we change this? Did options also suffer from the same issue as GameConfig?
Changing all those dots to arrows must have been a hassle.